pkg/kubelet/rkt/config.go

```diff
@@ -16,7 +16,13 @@ limitations under the License.
 
 package rkt
 
-import "fmt"
+import (
+	"flag"
+	"fmt"
+	"strings"
+)
+
+const DefaultOptions = "--dir=/var/lib/rkt --insecure-options=image,ondisk --local-config=/etc/rkt"
```

Comment:
@alban I removed the 2nd commit so that this PR doesn't depend on #21718 Will update on that one.